pylama.ini

```diff
@@ -1,5 +1,6 @@
 [pylama]
 format = mccabe,pep8,pep257,pyflakes,pylint
+skip = tests
 ignore = C901
```

Comment:
It looks like that pylama automatically checks for `tox.ini` too, so maybe we can simply combine those two ini files.